### PR TITLE
Update config.yml to run sample app UI tests

### PR DESCRIPTION
**Pull request by desokroshan**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -141,6 +141,26 @@ jobs:
...
+      xcode: "10.1.0"
+    steps:
+      - checkout
+      - run:
+          name: Run Sample UI Test
+          command: |
+            git clone https://github.com/awslabs/aws-sdk-ios-samples.git
+            cfolder=$(pwd)
+            cd aws-sdk-ios-samples
+            cd S3TransferUtility-Sample/Swift/
```
Comment (rohandubal) [inline .circleci/config.yml]: can be combined with above `cd` command

File: .circleci/config.yml [hunk 2]
```diff
@@ -141,6 +141,26 @@ jobs:
...
+    # Specify the Xcode version to use
+    macos:
+      xcode: "10.1.0"
+    steps:
+      - checkout
+      - run:
+          name: Run Sample UI Test
+          command: |
+            git clone https://github.com/awslabs/aws-sdk-ios-samples.git
+            cfolder=$(pwd)
```
Comment (rohandubal) [inline .circleci/config.yml]: can we name it something like `rootDirectory`?

File: .circleci/config.yml (hunk 1 above)
Comment (desokroshan) [reply .circleci/config.yml]: Done

File: .circleci/config.yml (hunk 2 above)
Comment (desokroshan) [reply .circleci/config.yml]: Renamed it appropriately.

**Comment by desokroshan** [comment]
Releasing it via a different path. Closing this PR

**Pull request closed without merging**